jdaviz/configs/cubeviz/plugins/tools.py

```diff
@@ -41,8 +41,7 @@ def on_mouse_event(self, data):
             # throttle to 200ms
             return
 
-        msg = SliceSelectWavelengthMessage(wavelength=data['domain']['x'], sender=self)
```

Comment:
Do we even need `SliceSelectWavelengthMessage` with this removal? Should we remove this message from the code base altogether?

Reply:
@kecnry , any reason we still need to listen to `SliceSelectWavelengthMessage` in app at all?

Reply:
Quickly looking through the code, and it looks like the slider _was_ the only thing that used it? I'll remove it

Reply:
Agreed, if its not used anywhere else, let's get rid of it here.